### PR TITLE
Add new attributes for dangling lines expert filter

### PR DESCRIPTION
**Pull request by EtienneLt**
add attributes pairing key and tie line id to dangling line expert filters

### CODE REVIEW

File: src/components/translations/filter-expert-en.ts [hunk 1]
```diff
@@ -178,6 +178,8 @@ const filterExpertEn = {
     shuntSusceptance1: 'Shunt susceptance 1 (μS)',
     shuntConductance2: 'Shunt conductance 2 (μS)',
     shuntSusceptance2: 'Shunt susceptance 2 (μS)',
+    pairingKey: 'Pairing key',
```
Comment (thangqp) [inline src/components/translations/filter-expert-en.ts:181]:
```suggestion
    pairingKey: 'UCTE Xnode',
```

File: src/components/translations/filter-expert-en.ts [hunk 2]
```diff
@@ -178,6 +178,8 @@ const filterExpertEn = {
...
+    tieLineId: 'Tie line id',
```
Comment (thangqp) [inline src/components/translations/filter-expert-en.ts:182]:
```suggestion
    tieLineId: 'Tie line ID',
```

File: src/components/translations/filter-expert-fr.ts [hunk 3]
```diff
@@ -177,6 +177,8 @@ const filterExpertFr = {
     shuntSusceptance1: 'Susceptance parallèle 1 (μS)',
     shuntConductance2: 'Conductance parallèle 2 (μS)',
     shuntSusceptance2: 'Susceptance parallèle 2 (μS)',
+    pairingKey: "clé d'appareillement",
```
Comment (thangqp) [inline src/components/translations/filter-expert-fr.ts:180]:
```suggestion
    pairingKey: 'Xnode UCTE',
```

File: src/components/translations/filter-expert-fr.ts [hunk 4]
```diff
@@ -177,6 +177,8 @@ const filterExpertFr = {
...
+    tieLineId: "Id de l'interconnexion",
```
Comment (thangqp) [inline src/components/translations/filter-expert-fr.ts:181]:
```suggestion
    tieLineId: "ID de l'interconnexion",
```

File: src/components/translations/filter-expert-fr.ts (hunk 3 above)
Comment (EtienneLt) [reply src/components/translations/filter-expert-fr.ts:180]: are you sure ? because it is the deprecated name

Comment (thangqp) [reply src/components/translations/filter-expert-fr.ts:180]: See with Christelle..

**Review by thangqp — APPROVED** [review]

**Review by ghazwarhili — APPROVED** [review]

**Pull request merged**